src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -46,7 +47,10 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<HttpExceptionMiddleware>();
             builder.UseMiddleware<ResponseBufferingMiddleware>();
             builder.UseMiddleware<HomepageMiddleware>();
-            builder.UseMiddleware<HttpThrottleMiddleware>();
+            builder.UseWhen(context => context.Features.Get<IFunctionExecutionFeature>() != null, config =>
+            {
```

Comment:
Why this change?

Reply:
If the JobHost wasn't running (or was waiting due to an error and was in its backoff retry), we were unable to hit admin routes b/c this Middleware needed the HttpRequestQueue, which only comes from the JobHost. @fabiocav and I thought that this was an okay approach since we hadn't want to throttled on admin routes previously anyway.

Reply:
Yeah, this gets us back to the previous behavior, where only functions are throttled